### PR TITLE
Docs: moved introduction from readme to docs folder

### PR DESCRIPTION
**Pull request by DesmondSanctity**
##### SUMMARY
Moved *Introduction* from ReadME to it's folder in /docs directory. 

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### ADDITIONAL INFORMATION
This PR is in line with the docs revamp and tracked by issue #1360

### CODE REVIEW

**Comment by samccann** [comment]
Part of https://github.com/ansible/awx-operator/issues/1360

File: README.md [hunk 1]
```diff
@@ -14,7 +14,6 @@ An [Ansible AWX](https://github.com/ansible/awx) operator for Kubernetes built w
 <!--ts-->
 * [AWX Operator](#awx-operator)
```
Comment (samccann) [inline README.md:18]: Note: we are in the process of moving this readme into official docs in the /docs folder. Please go there to find additional sections during this interim move phase.

**Comment by samccann** [comment]
@DesmondSanctity I was going to try rebasing these PRs and pushing my commit back up to each to move this project along.

**Comment by DesmondSanctity** [comment]
> @DesmondSanctity I was going to try rebasing these PRs and pushing my commit back up to each to move this project along.

That will be okay . Waiting for the push

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Ok this worked. I'll go through the other docs PRs one at a time to rebase and merge. thanks!

**Pull request merged**